src/mixer.c

```diff
@@ -379,6 +379,65 @@ static void PrintFormat(char *title, SDL_AudioSpec *fmt)
 }
 #endif
 
+/*
+	Initialize the Mixer internals (channels, chunk and music decoders)
+	with an existing AudioSpec, without taking over the callback.
+
+	Allows the calling to use their spec and audio callback.
+	The caller must manage AudioOpen and CloseAudio externally.
+*/
+int SDLCALLCC Mix_InitMixer(const SDL_AudioSpec spec)
+{
+    /* If the mixer is already initialized, increment open count */
+    if (audio_opened) {
+        if (spec.format == mixer.format && spec.channels == mixer.channels) {
+            ++audio_opened;
+            return(0);
+        }
+        while (audio_opened) {
+            Mix_FreeMixer();
+        }
+    }
+
+	mixer = spec;
```

Comment:
**Tabs** :vs: **Spaces**
*Fight!* :boxing_glove: 

Original SDL Mixer and SDL Mixer X are preferring 4-spaces, it's better to don't mix tabs and spaces as there are making ugly result on different setup of tab size. Be careful, dude! :fox_face: :wink:

Reply:
Fixed in the queued pull request.